ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__installer-rehearse-4.21.yaml

```diff
@@ -139,6 +139,19 @@ tests:
     env:
       EXTRACT_MANIFEST_INCLUDED: "true"
     workflow: cucushift-installer-rehearse-azure-stack-upi
+- as: installer-rehearse-aws-gp3-throughput
```

Comment:
And for regular CI testing, we can add this test point as a new job like https://github.com/openshift/release/blob/master/ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__installation-nightly-4.21.yaml#L317-L333 in `openshift-verification-tests-main__installation-nightly-4.21.yaml`